include/GafferSceneUI/StandardLightVisualiser.h

```diff
@@ -69,25 +69,46 @@ class GAFFERSCENEUI_API StandardLightVisualiser : public IECoreGLPreview::LightV
 
 	protected :
 
-		static const char *faceCameraVertexSource();
```

Comment:
@danieldresser-ie We had a poke in the IE code base, and this seemed to be the only change that should affect you. Are you ok to duplicate this shader for now?

Reply:
I've merged this PR so Tom can keep moving more easily, but we can definitely bring this method back if needed. We just felt that it was so hopeless opaque that it was best off removed...